TypeCobol.Test/TestCollection.cs

```diff
@@ -11,12 +11,14 @@
 using TypeCobol.Test.Parser.Text;
 using TypeCobol.Test.Utils;
 
-namespace TypeCobol.Test {
+namespace TypeCobol.Test
+{
 
     [TestClass]
     public class TestCollection
     {
         static readonly string root = PlatformUtils.GetPathForProjectFile("Parser" + Path.DirectorySeparatorChar + "Programs");
+        static readonly string rootNew = PlatformUtils.GetPathForProjectFile("Parser" + Path.DirectorySeparatorChar + "ProgramsNew");
```

Comment:
Why create a new folder for programs ?

Reply:
Because I use a new kind of dump that uses new Semantic Domain Symbol Type ToString. Thus based of this new folder I can use this new dump style, and let current dump style as it is.

Reply:
In this case, you must use a new Comparator instead of a folder.
As stated by @fm-117, the SYM comparator seems to be what you need.